content/tags/ai.yml

```diff
@@ -0,0 +1,2 @@
+name: Artificial Intelligence
+description: Artificial Intelligence (AI) is the ability for computer systems to perform tasks that would normally require human intelligence
```

Comment:
@VickyGarnett, how about: Artificial Intelligence (AI) is a branch of computer science aimed at developing computer systems that can solve complex problems by simulating human cognitive functions

Reply:
Even better, thanks @ttasovac